doc/SkillClaimsValidation.md

```diff
@@ -0,0 +1,42 @@
+# HowTo: Block all Skill Claims
+
+Add a class inheriting from `ClaimsValidator` which overrides `ValidateClaimsAsync` and throws an exception if the claims are skill claims:
+```cs
+    public class NoSkillsClaimsValidator : ClaimsValidator
+    {
+        public override Task ValidateClaimsAsync(IList<Claim> claims)
+        {
+            if (SkillValidation.IsSkillClaim(claims))
+            {
+                throw new UnauthorizedAccessException("Invalid call from a skill.");
+            }
+            return Task.CompletedTask;
+        }
+    }
+```
+
+Update `BotFrameworkHttpAdapter` implementation, to pass the base constructor an `AuthenticationConfiguration` using the `NoSkillsClaimsValidator`:
+
+```cs
+    public class AdapterWithErrorHandler : BotFrameworkHttpAdapter
+    {
+        //public AdapterWithErrorHandler(IConfiguration configuration, ILogger<BotFrameworkHttpAdapter> logger)
+        //    : base(configuration, logger)
```

Comment:
Does including the old `base()` call add any benefit? I think it could be removed without issue.

Reply:
Yes, this can be removed.  I left it for clarity of what was modified, but i'll remove it as both you and josh don't like it :)